nss-day-cohort-44/nutshell-hunting-hagravens
# Jl add event button

# Description
Created 'addEventButton' class inside index.html div. Added EventButton.js component and created button with innerHTML method. renderNewEventButton function is exported to Nutshell.js.

Fixes # (issue)
## Type of change


# Testing Instructions
`git fetch --all`
`git checkout jl-add-event-button`
Should display button inside events box.

# Checklist:
